src/tactics/Unit.js

```diff
@@ -92,7 +92,7 @@ export default class Unit {
     if (this.aLOS === true)
       return this.getLOSTargetTiles(target, source);
     else if (this.aAll === true)
-      return this.getAttackTiles(source);
+      return [...this.getAttackTiles(source), this.assignment];
```

Comment:
To deduplicate, you can use this line:
`return [ ...new Set([ ...this.getAttackTiles(source), this.assignment ]) ];`